src/main/java/me/zeroeightsix/kami/module/modules/misc/AutoObsidian.kt

```diff
@@ -401,7 +416,6 @@ object AutoObsidian : Module() {
                 MessageSendHelper.sendChatMessage("$chatName No ender chest was found in inventory, disabling.")
                 mc.soundHandler.playSound(PositionedSoundRecord.getRecord(SoundEvents.ENTITY_EXPERIENCE_ORB_PICKUP, 1.0f, 1.0f))
                 disable()
-                return
```

Comment:
I wasn't sure about this. If we call disable() do we still need to return? Or will disabling cause the method to return early?

Reply:
No, disabling does not cause the method to return.